src/cmd/tools/crane.go

```diff
@@ -88,3 +91,54 @@ func zarfCraneCatalog(cranePlatformOptions *[]crane.Option) *cobra.Command {
 
 	return craneCatalog
 }
+
+// Wrap the original crane list with a zarf specific version
+func zarfCraneList(cranePlatformOptions *[]crane.Option) *cobra.Command {
+	craneList := craneCmd.NewCmdList(cranePlatformOptions)
+
+	eg := `  # list the tags for a repo internal to Zarf
+  $ zarf tools registry ls internal/stefanprodan/podinfo
+
+  # list the tags for a repo hosted at reg.example.com
+  $ zarf tools registry ls reg.example.com/stefanprodan/podinfo`
+
+	craneList.Example = eg
+	craneList.Args = nil
+
+	originalListFn := craneList.RunE
+
+	craneList.RunE = func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			message.Fatal(nil, lang.CmdToolsCraneListNoRepoSpecified)
+		}
+
+		if !strings.HasPrefix(args[0], "internal/") {
+			return originalListFn(cmd, args)
+		}
+
+		// Load Zarf state
+		zarfState, err := cluster.NewClusterOrDie().LoadZarfState()
```

Comment:
Instead of `cluster.NewClusterOrDie()` we will need to use `cluster.NewCluster()` and then manually check if we are connected to the cluster.

Right now there is no way to call the original `crane list` function without having an initialized k8s cluster.

Reply:
Can you clarify the 2nd part of your comment?  You run `crane list` against a registry, not a k8s cluster.  Are you speaking specifically about a Zarf initialized cluster?

Reply:
I believe I understood what you said.  Please review my latest changes.